src/main/java/racingcar/dto/Input.java

```diff
@@ -15,9 +15,18 @@ private Input(List<String> namesOfCars, int numberOfCountToTry) {
     }
 
     public List<RacingCar> toRacingCars() {
-        return namesOfCars.stream()
+        List<RacingCar> racingCars = namesOfCars.stream()
                 .map(RacingCar::new)
                 .collect(Collectors.toList());
+
+        validateRacingCars(racingCars);
+        return racingCars;
```

Comment:
해당 메서드도 반환값인 컬렉션을 변경할 수 없도록 변경해주시면 좋을 것 같아요 :)
```suggestion
        return Collections.unmodifiableList(racingCars);
```

Reply:
해당 list는 이후에 Collections.sort(racingCars); 과정이 필요해서 unmodifiable 적용이 불가능 하더라구요 ㅠ